soapui/src/main/java/com/eviware/soapui/impl/rest/panels/request/views/html/HttpHtmlMessageExchangeResponseView.java

```diff
@@ -232,8 +232,8 @@ public void setEditable(boolean enabled) {
     }
 
     @Override
-    public boolean supportsContentType(String contentType ) {
-        return contentType.toLowerCase().endsWith("html");
+    public int getSupportScoreForContentType(String contentType ) {
+        return contentType.toLowerCase().endsWith("html")? 2 : 0;

```

Comment:
I would prefer to see some named consts instead of 0 and 2.

Reply:
It's just a score, like we have in EventBus event priorities. Maybe I can just explain this in Javadoc?

Reply:
yes, please

Reply:
fixed